### PR TITLE
Replace CHANGELOG.md with NEWS.md similar to other shiny.tools packages

### PR DESCRIPTION
**Pull request by anirbanshaw24**
# Add NEWS.md file

1. Adds NEWS.md file following guidelines in the [clickup task](https://app.clickup.com/2585644/inbox?tab=important)
2. WORDLIST in case agnostic alphabetical order

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+- Changed the dots (`...`) argument in `router_ui()` to allow dynamically passed arguments
+- Fixed the issue where the 404 page was not working when a non-valid page is opened as the first page
+
+# shiny.router 0.3.0
+
+- Added a new API that is compatible with Rhino
+- `router_ui` and `router_server` replaced `make_router`
+- `router_ui` should be applied inside the UI function (not outside, like `make_router`) and thanks to that, can utilize `ns()` if used in a Rhino app
+- `router_server` adds all required server components (mostly `observeEvents`) to the server part of the application
+- Marked `make_router` as deprecated, it is not removed
```
Comment (jakubnowicki) [inline NEWS.md:12]: 1. It suggests that introducing `router_ui` and `router_server` is something different than the new API. This has to be a single point. Don't add so many low-level technical details. It should include information that there is a new API compatible with Rhino (you can mention function names) and the old one is deprecated.

File: NEWS.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+# shiny.router 0.2.3
+
+- Applied `shiny::createWebDependency()` before `renderDependencies()`
```
Comment (jakubnowicki) [inline NEWS.md:16]: This is not a good description - this informs how the bug was solved, but tells nothing about what the bug was. Again, you need to read the PRs - in this particular case, you would find the link to the issue with bug description. Based on that, create an entry for the changelog.

File: NEWS.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# shiny.router (development version)
```
[the remark this answers is not in the document]
Comment (anirbanshaw24) [reply NEWS.md:1]: Alright. So even though this version is not present in CRAN, we have it here as its a version on github right ?

File: inst/WORDLIST
[the remark this answers is not in the document]
Comment (anirbanshaw24) [reply inst/WORDLIST:1]: Okay but want to know the reason. Case agnostic is not required ?

[the remark this answers is not in the document]
Comment (jakubnowicki) [reply inst/WORDLIST:1]: Yes, they should be at the top. `spelling` package orders them that way and there is no point in changing that. Besides, this pattern was already there. In such a case, when you are working on something else, ask first.

[the remark this answers is not in the document]
Comment (anirbanshaw24) [reply inst/WORDLIST:1]: Alright. 👍🏼

**Review by jakubnowicki — APPROVED** [review]
Please resolve the comment and you can merge. Please use "squash and merge" option to avoid flooding the git history with unnecessary commits.

**Pull request merged**